### PR TITLE
Fix: False Positives in the dependencies check

### PR DESCRIPTION
**Pull request by y0urself**
**What**:

**Why**:

**How**:

**Checklist**:

- [ ] Tests
- [ ] Conventional commit message
- [ ] Documentation

### CODE REVIEW

File: troubadix/plugins/dependencies.py [hunk 1]
```diff
@@ -72,7 +74,9 @@ def run(
                     if "+d+.nasl" in dep:
                         continue
 
-                    if not (root / dep).exists():
+                    if not any(
+                        (root / vers / dep).exists() for vers in FEED_VERSIONS
+                    ):
```
Comment (pascalholthaus) [inline troubadix/plugins/dependencies.py:79]: Maybe I'm wrong, but when the feed is generated it's all in one directory called nasl. Maybe check (root / dep) too?

Comment (y0urself) [reply troubadix/plugins/dependencies.py:79]: i thought about this, but this script is not running there, I think?

Comment (pascalholthaus) [reply troubadix/plugins/dependencies.py:79]: At least on every PR it is running on a generated feed release.

File: troubadix/plugins/dependencies.py [hunk 2]
```diff
@@ -31,6 +31,8 @@
     LinterWarning,
 )
 
+FEED_VERSIONS = ["common", "21.04", "22.04"]
```
Comment (y0urself) [inline troubadix/plugins/dependencies.py:34]:
```suggestion
FEED_VERSIONS = ["common", "21.04", "22.04", ""]
```

File: troubadix/plugins/dependencies.py (hunk 1 above)
Comment (y0urself) [reply troubadix/plugins/dependencies.py:79]: I have added `""` ;)

**Review by pascalholthaus — APPROVED** [review]

**Pull request merged**